src/appleseed-max-impl/utilities.h

```diff
@@ -54,6 +54,7 @@
 #include <maxtypes.h>
 #include <point3.h>
 #include <point4.h>
+#undef base_type
```

Comment:
Fix here too.

In fact, it seems to me that there are many other places where Max files are included but are missing the header guards.

Reply:
I'm adding guards in .h files. Should I do the same for .cpp? Looks like it, they also include max headers _after_ their own .h files..

Reply:
But because of `#pragma once` it won't do anything in .cpp file, right? Should I remove `#pragma` statement?